src/main/kotlin/blackjack/model/Suit.kt

```diff
@@ -0,0 +1,8 @@
+package blackjack.model
+
+enum class Suit(val symbol: Char) {
+    HEARTS('♥'),
+    DIAMONDS('♦'),
+    CLUBS('♣'),
+    SPADES('♠'),
+}
```

Comment:
When the Blackjack app convert to Web browser image game,
fellow develop should check the `InputView`, `OutputView`, `Card`, ... a lot of classes.

How about move all view logic to View classes?
Or, u can make duplicate enum class in View class file only used for output on terminal.

```kotlin
enum class SuitView(
    val suit: Suit,
    val symbol: String,
) {
    HEART(suit = Suit.HEART, symbol = "♥"),
    // ...
}
```